firedrake/plot.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+
+
+def one_dimension_plot(function, num_points):
```

Comment:
Ask discussed over lunch, I think we should call this `plot`.  We will also make a matplotlib figure and return it.  So the signature will be:

```python
def plot(function, num_points, **kwargs):
   """Plot a function and return a matplotlib figure object.

   :arg function: The function to plot.
   :arg num_points: Number of points per element
   :arg kwargs: Additional keyword arguments passed to ``matplotlib.plot``.
   """
   ...
   fig = plt.plot(..., **kwargs)
   return fig
```